### ISSUE TITLE
Possibility to add links and issues template

### ISSUE THREAD

**Issue by swallow66** [Noksa/Allure.NUnit#18]
Is there a plan to support link/defect/tms templates in allureConfig.json(like a main version of Allure report)? 

Currently the following error occurs:
Message: OneTimeSetUp: Newtonsoft.Json.JsonSerializationException : Unexpected token when deserializing object: StartArray. Path 'allure.links', line 8, position 3.

allureConfig.json
{
	"allure": {
		"directory": "allure-results",
		"allowEmptySuites": true,
		"enableParameters": true,
		"allowLocalHistoryTrend": false,
		"links": //optional
		[
			"https://example.org/{link}",
			"https://example.org/{issue}",
			"https://example.org/{tms}"
		]
	}
}

**Comment by Noksa** [comment]
Hey.
Yes, in the near future. :)

**Comment by Noksa** [comment]
@swallow66 

This feature available since 2.5.1 version.

In addition, a new boolean optional argument has been added to tms/issue/link attributes, which responsible for replacement by pattern.
If true - replace will be done.
If false - not.

Example:
```
[AllureLink("http://mysite.com/ID-123", false)] // will not replaced by pattern
[AllureLink("ID-124")] // will replaced
```
